pages/04.HDM Doc 2.2/18.appendix/default.md

```diff
@@ -170,54 +170,95 @@ The document describes the interactions between HDM components and VMware Infras
+| CentOS        | CentOS 8.2                      | UEFI Secure | Not Supported          | Not Supported          | Qualified           | Qualified                    | Not Supported                |
+| CentOS        | CentOS 8.3                      | MBR         | Qualified              | Qualified              | Qualified           | Qualified                    | Qualified                    |
+| CentOS        | CentOS 8.3                      | UEFI        | Qualified              | Qualified              | Qualified           | Qualified                    | Qualified                    |
+| CentOS        | CentOS 8.3                      | UEFI Secure | Not Supported          | Not Supported          | Qualified           | Qualified                    | Not Supported                |
+| Ubuntu        | Ubuntu Server 16.04             | MBR         | Qualified              | Qualified              | Qualified           | Qualified                    | Qualified                    |
+| Ubuntu        | Ubuntu Server 16.04             | UEFI        | Not Supported          | Not Supported          | Qualified           | Qualified                    | Not Supported                |
+| Ubuntu        | Ubuntu Server 16.04             | UEFI Secure | Not Supported          | Not Supported          | Qualified           | not qualified(CP-6117)       | Not Supported                |
+| Ubuntu        | Ubuntu Server 18.04             | MBR         | Qualified              | Qualified              | Qualified           | Qualified                    | Qualified                    |
+| Ubuntu        | Ubuntu Server 18.04             | UEFI        | not qualified(CP-5956) | Qualified              | Qualified           | Qualified                    | not qualified(CP-5956)       |
+| Ubuntu        | Ubuntu Server 18.04             | UEFI Secure | Not Supported          | Not Supported          | Qualified           | Qualified                    | Not Supported                |
+| Ubuntu        | Ubuntu Server 20.04             | MBR         | Qualified              | Qualified              | Qualified           | Qualified (Need SI handling) | Qualified                    |
+| Ubuntu        | Ubuntu Server 20.04             | UEFI        | Qualified              | Qualified              | Qualified           | Qualified (Need SI handling) | Qualified                    |
+| Ubuntu        | Ubuntu Server 20.04             | UEFI Secure | Not Supported          | Not Supported          | Qualified           | Qualified (Need SI handling) | Not Supported                |
+| Oracle\_Linux | Oracle\_Linux 6.8               | MBR         | not qualified(DP-2962) | not qualified(DP-2962) | Qualified           | Qualified                    | not qualified(DP-2962)       |
```

Comment:
_not -> Not_
Also keep the **(** consistent. In the rows above this there is space before (. But here it is not.

Reply:
Resolved